straxen/plugins/acqmon_processing.py

```diff
@@ -240,6 +242,160 @@ def compute(self, aqmon_hits, start, end):
         result = result[sort]
         return result
 
+@export
+class GPS_sync(strax.Plugin):
+    """
+    Correct the event times to GPS time. 
+      1. Finds the TTL GPS pulses coming into the AM from the gps 
+    module and their pairs in the ASCII file coming from the module
```

Comment:
There are no ASCII files now, correct? Everything's been ported into the database?

Reply:
Yes. I'll update the docstring.